openwisp_firmware_upgrader/migrations/0013_alter_upgradeoperation_progress.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.2.4 on 2025-07-24 19:36
```

Comment:
can you squash this migration with the previous one please? See https://docs.djangoproject.com/en/5.2/topics/migrations/#migration-squashing